modules/sc-mesh-secure-deployment/src/nats/src/cbma_adaptation.py

```diff
@@ -856,8 +856,6 @@ def stop_cbma(self) -> None:
         Stops CBMA by terminating CBMA processes. Also
         destroys batman and bridge interfaces.
         """
-        if not self.__cbma_set_up:
```

Comment:
I believe `self.__cbma_set_up` is now unused in CBMAAdaptation, plz remove it if so :2nd_place_medal:

Reply:
Removed in commit 1826923e3d821a8ab2c2b156ce5c2d2db02a0359.